rudder-web/src/main/scala/bootstrap/liftweb/checks/CheckDSCSystemPolicy.scala

```diff
@@ -0,0 +1,135 @@
+        new Attribute("objectClass", "nodeGroup", "top")
+      , new Attribute("cn", "All nodes with a Rudder Windows DSC agent")
+      , new Attribute("isDynamic", TRUE.toLDAPString)
+      , new Attribute("nodeGroupId", "dsc-based-agent")
+      , new Attribute("description", "All nodes with a Rudder Windows DSC agent")
+      , new Attribute("isEnabled", TRUE.toLDAPString)
+      , new Attribute("isSystem", TRUE.toLDAPString)
+      , new Attribute("jsonNodeGroupQuery", """{"select":"nodeAndPolicyServer","composition":"And","where":[
+                                                   {"objectType":"node","attribute":"policyServerId","comparator":"eq","value":"root"}
+                                               ]}""")
+    )
+
+    val dscCommonTechnique = LDAPEntry(new DN("activeTechniqueId=dsc-common,techniqueCategoryId=Rudder Internal,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration"),
+        new Attribute("objectClass", "activeTechnique", "top")
+      , new Attribute("acceptationTimestamp", """{"1.0":"20170101010100.000Z"}""")
+      , new Attribute("activeTechniqueId", "dsc-common")
+      , new Attribute("techniqueId", "dsc-common")
+      , new Attribute("isEnabled", TRUE.toLDAPString)
+      , new Attribute("isSystem", TRUE.toLDAPString)
+    )
+
+    val dscCommonDirective = LDAPEntry(new DN("directiveId=dsc-common-all,activeTechniqueId=inventory,techniqueCategoryId=Rudder Internal,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration"),
+        new Attribute("objectClass", "directive", "top")
+      , new Attribute("directiveId", "dsc-common-all")
+      , new Attribute("techniqueVersion", "1.0")
+      , new Attribute("cn", "DSC Based agent configuration")
+      , new Attribute("description", "System configuration for base agent")
+      , new Attribute("directivePriority", "0")
+      , new Attribute("isEnabled", TRUE.toLDAPString)
+      , new Attribute("isSystem", TRUE.toLDAPString)
+      , new Attribute("serializedTags", "[]")
+    )
+
+    val dscCommonRule = LDAPEntry(new DN("ruleId=dsc-agent-all,ou=Rules,ou=Rudder,cn=rudder-configuration"),
+        new Attribute("objectClass", "rule", "top")
+      , new Attribute("ruleId", "dsc-agent-all")
+      , new Attribute("serial", "0")
+      , new Attribute("cn", "Rudder system policy: base configuration for DSC based agent")
+      , new Attribute("description", "Rudder system policy: base configuration for DSC based agent")
+      , new Attribute("longDescription", "This rule manage based configuration for DSC agent")
+      , new Attribute("ruleTarget", "group:dsc-based-agent")
+      , new Attribute("directiveId", "dsc-common-all")
+      , new Attribute("isEnabled", TRUE.toLDAPString)
+      , new Attribute("isSystem", TRUE.toLDAPString)
+    )
+
+    val all = dscGroup :: dscCommonTechnique :: dscCommonDirective :: dscCommonRule :: Nil
```

Comment:
Because I don't see any reason to do so. Do you see any? It is system code, it can belongs to code and that make one less file to look into and synchronise and everything. Or do you think there is a use case for the ldif outside of that part of code?